### DISCUSSION TITLE
new error when trying to play back event clips

### DISCUSSION THREAD

**Discussion by rnaff** [blakeblackshear/frigate#3740] [Ask A Question] [answered]
hello frigate community !!

I just started getting this error when trying to play back clips, never had an issue playing clips in the past.

"The media could not be loaded, either because the server or network failed or because the format is not supported"

I'm not sure if there was an update and I need to change something around?

Thanks in advance.

**Comment by NickM-27** [comment]
What version of frigate are you running? What do the browser logs say about the error? Do you see frigate logs?

**Comment by rnaff** [comment, marked as answer]
I just checked the logs and it's cuz I'm out of disk space hahaha I'm going
to delete this,

Thanks for replying

**Reply by NickM-27** [reply]
No need to delete, someone may run into the same question
